rebasehelper/application.py

```diff
@@ -482,7 +482,7 @@ def build_source_packages(self):
 
         for version in ['old', 'new']:
             koji_build_id = None
-            results_dir = '{}-build'.format(os.path.join(self.results_dir, version))
+            results_dir = os.path.join('{}-build'.format(os.path.join(self.results_dir, version)), 'SRPM')
```

Comment:
This is ugly and unnecessary.
Why not simply `os.path.join(self.results_dir, '{}-build'.format(version), 'SRPM')`?